project/docker/Dockerfile

```diff
@@ -1,10 +1,34 @@
 FROM alpine
 MAINTAINER Benedict Lau <benedict.lau@groundupworks.com>
 
-RUN apk add --no-cache nodejs
+RUN apk add --no-cache nodejs nginx php5-fpm php5-json socat
 
 RUN apk add --no-cache --virtual .build libtool nodejs-npm autoconf automake build-base python &&  \
-    npm install scuttlebot -g --unsafe-perm ; \
+    npm install scuttlebot \
+                ssb-keys \
+                ssb-client \
+                ssb-feed \
+                pull-stream \
+                    -g --unsafe-perm ; \
     apk del .build
 
-ENTRYPOINT sbot server
+COPY nginx.conf /etc/nginx/conf.d/default.conf
+
+RUN mkdir /run/nginx; \
+    apk add --no-cache --virtual .git git; \
+    git clone https://github.com/darkdrgn2k/ssb-web-pi.git ~/ssb-web-pi.git; \
```

Comment:
Can we remove this `.git` from the filename? Generally when we clone a repo, the default is not to have the `.git` extension in the path.

Reply:
Fixed